LibGUI/GVariant.h

```diff
@@ -194,7 +194,7 @@ class GVariant {
         RawPoint as_point;
         RawSize as_size;
         RawRect as_rect;
-    } m_value; 
+    } m_value;
```

Comment:
This is an unrelated change and should not be in this commit.

Reply:
This was automatically done by my editor, I don't think it's worth excluding.

Reply:
Okay for this time, but in the future I'd prefer to not touch unrelated files since that makes the git history unnecessarily confusing :)